eldritch/encounters.py

```diff
@@ -144,11 +144,101 @@ def Society4(char):
   cond = events.Conditional(char, check, "successes", {0: events.Nothing(), 2: skill})
   return events.Sequence([check, cond], char)
 
+def Administration1(char):
+  check = events.Check(char, "lore", -1)
+  dollars = events.Gain(char, {"dollars": 5})
+  return events.PassFail(char, check, dollars, events.Nothing())
+def Administration2(char):
+  return events.Gain(char, {"clues": 1})
+def Administration3(char):
+  check =  events.Check(char, "will", -1)
+  retainer = events.StatusChange(char, "retainer")
+  return events.PassFail(char, check, retainer, events.Nothing())
+def Administration4(char):
+  check = events.Check(char, "lore", -2)
+  spell = events.Draw(char, "spells", 1)
+  curse = events.Curse(char)
+  return events.PassFail(char, check, spell, curse)
+def Administration5(char):
+  return events.ForceMovement(char, "Asylum") # TODO: also have an event
+def Administration6(char):
+  # Administration 6 is identical to Administration 1
+  return Administration1(char)
 def Administration7(char):
   check = events.Check(char, "will", -2)
   gain = events.Gain(char, {"dollars": 8})
   arrest = events.Arrested(char)
   return events.PassFail(char, check, gain, arrest)
```

Comment:
While you're here, can you fix this? It should also give the player the choice of whether they want to make this check.

Reply:
done